chat/src/main/java/net/md_5/bungee/api/chat/SelectorComponent.java

```diff
@@ -33,6 +33,13 @@ public final class SelectorComponent extends BaseComponent
      */
     private String selector;
 
+    /**
+     * The separator of multiple entities
+     * the default is '{"color": "gray", "text": ", "}'
+     */
+    private BaseComponent[] separator;
```

Comment:
As per [cfe00fa47c](https://github.com/SpigotMC/BungeeCord/commit/cfe00fa47c19e71e191c2651deb61239d68ed18b) I think this should just be a simple `BaseComponent`, not an array